File: list_sort.py
#encoding: UTF8
import time,random

n = 100000
list = []
for i in range(n):
  list.insert(0,random.randint(1,n))
before_loop = time.clock()
for i in range(n):
  pass
after_loop = time.clock()
before = time.clock()
for i in range(n):
  list.sort()
  pass
print (time.clock()- before - (after_loop - before_loop))

File: list_sort.rb
# encoding: UTF-8

require "benchmark"
n = 100000
i = 0
list = Array.new
1.upto(n) {|i|
  list.insert(0,rand(i))
}
puts Benchmark::CAPTION
puts Benchmark.measure{
1.upto(n) {|i|
}
}
puts Benchmark.measure{
1.upto(n) {|i|
  list.sort!
}
}
